MangaTaggerLib/models.py

```diff
@@ -94,6 +97,12 @@ def _parse_genres(self, genres, logging_info):
             Metadata._log.debug(f'Genre found: {genre}')
             self.genres.append(genre)
 
+    def _parse_synonyms(self, synonyms, logging_info):
+        Metadata._log.info('Parsing Synonyms...', extra=logging_info)
+        for synonym in synonyms:
+            Metadata._log.debug(f'Synonym found: {synonym}')
+            self.synonyms.append(synonym)
+

```

Comment:
Taking a look at this....
Think of this as a suggestion/enhancement.
Couldn't we just print the list once?
and just do:
```suggestion
    def _parse_synonyms(self, synonyms, logging_info):
        Metadata._log.info('Parsing Synonyms...', extra=logging_info)
        Metadata._log.debug(f'Synonyms found: {", ".join(synonyms)}')
        self.synonyms = synonyms
        
```
Same would apply for genres just above

Reply:
I just copied what was implemented for genres.  Self.synonyms is an empty array at this point, the initial way may prevent it from being overwritten with None.  Not sure how relevant that is.

Reply:
self.synonym.extends(synonyms)  ?

Reply:
I guess that'd work? Also it being able to be none, the log statement world need to be changed. Just dumping the list could work. So itd be the same except without the join part

Reply:
I just though that if synonims is None extend would raise an exception.
Guess we can just leave it how it is. I have planned a rewrite, but it's still a long way off.

Reply:
I just run the test and i guessed right in my head. Was not sure what `anilist_details['synonyms']` and if it'd be safe. Change it to `anilist_details.get('synonyms')`

Reply:
I'm going to mark this as resolved and you can commit your changes for these _parse functions.
my testing with extend worked, however I did get one synonym with an extra space at the beginning.  So we could add a strip() if still looping through each synonym.